Scripts/Hooks/scratchpad-hook.lua

```diff
@@ -533,6 +535,14 @@ local function loadScratchpad()
         keyboardLocked = true
     end
 
+        local function runListeners(list)
+            for _, listener in pairs(list) do
+                if type(listener) == "function" then
+                        listener()
+                end
+            end
+        end
```

Comment:
Nit: indentation could be moved one level left

Reply:
Not sure when I fixed this indentation but it appears correct now.